docs/concepts/otlp/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: OTLP
```

Comment:
I'm a big proponent of including the full name next to abbreviation to make sure it's clear similar to [DSN](https://sentry-docs-git-georgegritsouk-ope-18-write-public-onboa-d1aec0.sentry.dev/concepts/key-terms/dsn-explainer/)
```suggestion
title: OpenTelemetry Protocol (OTLP)
```

Reply:
That suits me fine 👍🏻 Updated